supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -198,8 +205,25 @@ trait SharedFileSystem extends PathFactory {
       PairOfFiles(src, dest)
     }
 
+    // A possibly staged version of the input file suitable for downstream processing, or just the original input
+    // file if no staging was required.
+    val staged: WomFile = value.mapFile { input =>
+      pathBuilders.collectFirst({ case http: HttpPathBuilder if HttpPathBuilder.accepts(input) => http }) match {
```

Comment:
Could we multi-line this `case`?

Reply:
Right now it's a method on two of the path builder companion objects. I tried making it an instance method but IIRC the complexity of the rules on what's acceptable in GCS paths made me rethink that.

Reply:
Oops this reply probably makes more sense attached to your comment below. 🙂